news/7740.trivial

```diff
@@ -0,0 +1 @@
+Use PEP 517 layout by specifying ``build-backend``, so that ``pip`` can be built with tools (s.a. ``pep517``) that don't support the legacy layout.
```

Comment:
What's "s.a."?

Reply:
`such as`

Reply:
It's not (as far as I know) a well known abbreviation. Can you spell it out explicitly please?